Formula/helmfile.rb

```diff
@@ -0,0 +1,30 @@
+class Helmfile < Formula
+  desc "Deploy Kubernetes Helm Charts"
+  homepage "https://github.com/roboll/helmfile"
+  url "https://github.com/roboll/helmfile/archive/v0.18.0.tar.gz"
```

Comment:
There's a version update available now. The last CI build has been lost to churn so might as well push the update rather than asking for another rebuild manually.

Reply:
Just bumped to 0.19.0